src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -41,7 +34,7 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
     </jsp:attribute>
     <jsp:attribute name="bodyContent">
             <h1><%= request.getAttribute("customChatTitle") != null ? request.getAttribute("customChatTitle") : conversation.getTitle() %>
-              <a href="" style="float: right">&#8635;</a></h1>
+              <a href="" id="refresh">&#8635;</a></h1>
```

Comment:
I am applying this styling through this new id in the chat page's new stylesheet